### DISCUSSION TITLE
Rhoden and Sampley

### DISCUSSION THREAD

**Discussion by jsampley** [birkenkrahe/ai482#9] [Show and tell]
[Link to our presentation here](https://drive.google.com/file/d/1YUuQiFSYsqsOWjYS6DEUwfySJnQ8qfO-/view?usp=sharing)

**Comment by birkenkrahe** [comment]
Enjoyed! Probable made many of these comments before - as intended,
   the final presentation was leaning on previous prototype presentations.
   
   * The "perceived benefits/risks" graphs suggest the West/East split
     that seems to dominate politics. It would have been interesting
     to compare these with other world cultural data (e.g. the famous
     [[https://www.hofstede-insights.com/product/compare-countries/][Hofstede data sets]]). You'll find some of these results mirrored
     in the cultural value data. 
   * Facial recognition: it looks as if you've got to go to Cuba or
     Northern Africa to hide. Two places that one might not want to
     settle in for a different set of reasons...
   * Good on new for trying to crack this and even coming up with one
     settle in for a different set of reasons..application. Sorry that
     this course "broke your laptop" :-) 
   * ~ClarifAI~ was new to me - need to check it out.
   * Strong arguments for Linux here, nice.
   * Recognition and sentiment analysis - nice combo. Very different
     models though, I think. I wonder if sentiment is really
     stochastically relevant (as the prediction spectrum seems to
     suggest). I mean when I'm angry, I'm angry, and when I'm sad,
     then I'm sad. Assuming sentiment mixtures is based on a specific
     psychological model, so there's heavy bias here.
   * The croissant made me hungry. I thought it looked sad though.
   * You did good re: selecting topics. Coming out of the presentation
     enriched, with a few more, but more focused questions. Good reflection.
   * Jacob worked on cellular automata, and Spencer on robotics (in
     the past anyway?) - you labelled the necessary skills as
     "intermediate", so I'm not going to put this into an introductory
     course.
   * Video cut off at end.
